src/main/kotlin/com/chattriggers/ctjs/api/render/Image.kt

```diff
@@ -32,13 +32,7 @@ class Image(var image: BufferedImage?) {
 
     fun getTextureHeight(): Int = textureHeight
 
-    fun getTexture(): NativeImageBackedTexture {
-        requireNotNull(texture) {
-            "Failed to bake Image texture"
-        }
-
-        return texture!!.texture
-    }
+    fun getTexture(): NativeImageBackedTexture? = texture?.texture
```

Comment:
Is there a good reason not to put the `?: 0` here?

Reply:
The 0 refers to the glId, not the NativeImageBackedTexture

Reply:
oops I'm dumb